index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>OS X Calculator</title>
+</head>
+<meta charset='utf-8' />
+  <link rel='stylesheet' href='style.css'/>
+<body>
+
+  <div class='calculator'>
+    <div id='screen'> 0
```

Comment:
try to avoid using ids. Just use a unique classname. `.calculator-display` might be a good fit.